foreign_cc/meson.bzl

```diff
@@ -201,3 +233,11 @@ def meson_with_requirements(name, requirements, **kwargs):
         toolchain = full_label("built_meson_toolchain_for_{}".format(name)),
         **kwargs
     )
+
+# TODO: converge with cmake_script.bzl
```

Comment:
Is this TODO to be resolved as part of this PR?

Reply:
just did another experiment, we can just use the same with 
https://github.com/bazelbuild/rules_foreign_cc/blob/bae11c9a508953c1ec3a564102630878737de99d/foreign_cc/private/cmake_script.bzl#L313-L316

updated